[fpv] Adding VC Formal support to the script and document (#1737)
lowRISC/opentitan: doc/ug/dv_methodology.md

For professional tooling, the team has chosen [Synopsys' VCS](https://www.synopsys.com/verification/simulation/vcs.html) as the simulator of choice with respect to the tracking of verification completeness and [JasperGold](https://www.cadence.com/content/cadence-www/global/en_US/home/tools/system-design-and-verification/formal-and-static-verification/jasper-gold-verification-platform.html) for FPV.
Wherever possible we attempt to remain tool-agnostic, but we must choose a simulator as our ground truth for our own confidence of signoff-level assurances.

> For FPV, [VC Formal](https://www.synopsys.com/verification/static-and-formal-verification/vc-formal.html) is also supported with the same flow and its own setup files, that could be used for those who have VC formal licenses. 

Reviewer: Move this after line 26, and change to:

```suggestion
Likewise, for FPV, [Synopsys VC Formal](https://www.synopsys.com/verification/static-and-formal-verification/vc-formal.html) is also supported within the same flow, and can be used by those with access to VC Formal licenses. 
```
